### PR TITLE
Spellchecking

### PR DESCRIPTION
**Pull request by Findus23**
Thanks to the great open source spellchecker [languagetool](http://languagetool.org/) (which by the way are using Piwik for their analytics) I could quickly check all guides for typos and small style error.

Sorry for the deleted trailing whitespaces. I didn't notice my editor deleted them.

### CODE REVIEW

File: docs/3.x/tracking-javascript-guide.md [hunk 1]
```diff
@@ -764,36 +764,36 @@ As this solution causes every visitor's event, pageview, etc. to be tracked twic
...
 
-### Customise one of the tracker object instances 
+### Customise one of the tracker object instances
 
-Note: by default any tracker added via `addTracker` are configured the same as the main default tracker object (with regards to cookies, custom dimensions, user id, download & link tracking, domains and sub-domains, etc.). If you want to configure one of the Piwik tracker object instance that were added via `addTracker`, you may call the `Piwik.getAsyncTracker(optionalPiwikUrl, optionalPiwikSiteId)`  method. This method returns the tracker instance object which you can configure differently than the main JavaScript tracker object instance. 
+Note: by default any tracker added via `addTracker` are configured the same as the main default tracker object (regarding cookies, custom dimensions, user id, download & link tracking, domains and sub-domains, etc.). If you want to configure one of the Piwik tracker object instance that were added via `addTracker`, you may call the `Piwik.getAsyncTracker(optionalPiwikUrl, optionalPiwikSiteId)`  method. This method returns the tracker instance object which you can configure differently than the main JavaScript tracker object instance.
```
Comment (sgiehl) [inline docs/3.x/tracking-javascript-guide.md]: Not related to your changes, but shouldn't that be:

> Note: by default any tracker**s** added via `addTracker` are configured the same... 

or

> Note: by default any tracker added via `addTracker`  **is** configured the same...

and

> If you want to configure one of the Piwik tracker object instance**s** that **was** added via `addTracker` ```

File: docs/media-analytics/custom-player.md [hunk 2]
```diff
@@ -212,9 +212,9 @@ These HTTP Tracking API parameters can be used to track the usage of media:
 * `ma_le` - The duration (the length) of the media in seconds. For example if a video is 90 seconds long, the value should be `90`.  
 * `ma_ps` - The progress / current position within the media. Defines basically at which position within the total length the user is currently playing.
 * `ma_ttp` - Defines after how many seconds the user has started playing this media. For example a user might have seen the poster of the video for 30 seconds before a user actually pressed the play button.
-* `ma_w`  - The resolution width of the media in pixels. Only recommended to be set for videos.
-* `ma_h`  - The resolution height of the media in pixels. Only recommended to be set for videos.
-* `ma_fs` - Should be `0` or `1` and defines whether the media is currently viewed in full screen. Only recommended to be set for videos.
+* `ma_w`  - The resolution width of the media in pixels. Only recommended being set for videos.
+* `ma_h`  - The resolution height of the media in pixels. Only recommended being for videos.
```
Comment (sgiehl) [inline docs/media-analytics/custom-player.md]: the `set` was accidentally removed

**Comment by tsteur** [comment]
languagetool is great, I use them as well 👍

**Comment by Findus23** [comment]
@tsteur 
Then you can definitly use my command line client that supports checking markdown and HTML file:
https://github.com/Findus23/pyLanguagetool/

grafik

**Comment by tsteur** [comment]
👍  cheers

**Pull request merged**